boards/arm/nrf5340_pca10095/nrf5340_pca10095_cpu0_defconfig

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_NRF=y
+CONFIG_SOC_SERIES_NRF53X=y
+CONFIG_SOC_NRF5340_CPU0_QKAA=y
+CONFIG_BOARD_NRF5340_PCA10095_CPU0=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
```

Comment:
Is it a given that the user wants to enable the MPU?

Does this automatically protect certain areas?

I've seen issues where one image accidentally protected some area that the next image used.

Reply:
@SebastianBoe this is standard configuration, i.e. building ARM platforms with MPU support if MPU is present.

> Does this automatically protect certain areas?
When enabling MPU at build time, SRAM is configured as PRIV RW, and Code as RO. This is basic generic configuration i.e. regardless of User Mode, or anything else.

>I've seen issues where one image accidentally protected some area that the next image used.

I agree that we might need to do something about it. E.g. at boot, we could consider clening up all MPU programming left-out from previous image. There is a similar issue for Stack Pointer clean-up. I 'll see if i can enhance that issue to cover MPU clean-up.

Reply:
@SebastianBoe https://github.com/zephyrproject-rtos/zephyr/issues/7404